packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -9,11 +9,21 @@ import isEmpty from 'lodash/isEmpty';
 import map from 'lodash/map';
 import union from 'lodash/union';
 
+import { CDN_ACTIVE } from './hosting-multisite.constants';
+
+const CDN_STATISTICS_PERIOD = {
+  DAY: 'day',
+  MONTH: 'month',
+  WEEK: 'week',
+  YEAR: 'year',
```

Comment:
Can't `moment` handle those ?

Reply:
Do you mean if `moment` has its own enumerable with this date periods ?

Reply:
I'm having trouble understanding why we need those constants so I'm wondering if the use we have could be taken care of by `moment`

Reply:
Those constants are used to make API calls (filtering statistics period), and to set the chartjs xAxis `xMin`. It seems `moment` doesn't provide those kind of constants you referring to.